homeassistant/components/onewire/sensor.py

```diff
@@ -361,38 +355,6 @@ def get_entities(onewirehub: OneWireHub, config):
                 "Check the mount_dir parameter if it's defined"
             )
 
-    # We have an owfs mounted
-    else:  # pragma: no cover
-        # This part of the implementation does not conform to policy regarding 3rd-party libraries, and will not longer be updated.
-        # https://developers.home-assistant.io/docs/creating_platform_code_review/#5-communication-with-devicesservices
-        base_dir = config[CONF_MOUNT_DIR]
-        _LOGGER.debug("Initializing using OWFS %s", base_dir)
-        _LOGGER.warning(
-            "The OWFS implementation of 1-Wire sensors is deprecated, "
-            "and should be migrated to OWServer (on localhost:4304). "
-            "If migration to OWServer is not feasible on your installation, "
-            "please raise an issue at https://github.com/home-assistant/core/issues/new"
-            "?title=Unable%20to%20migrate%20onewire%20from%20OWFS%20to%20OWServer",
```

Comment:
Did we get any issue reports?

Reply:
The only issue I was made aware of was the one solved in #43599